buildpack.md

```diff
@@ -770,17 +777,23 @@ The stack `build-images` and `run-images` are suggested sources of images for pl
 [[processes]]
 type = "<process type>"
 command = "<command>"
+args = ["<arguments>"]
+direct = false
```

Comment:
should we make this a list of options? For instance, do we plan to support windows containers at some point?

Reply:
As mentioned during WG meeting:
All buildpack profile.d scripts are loaded when any process type runs, so many choices here leads to compatibility issues. Windows may have it's own meanings for `direct` (e.g., use powershell profile.d equivalent or skip it).